_posts/2025-01-09-v5-1-latest-release.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "v5.1.0 Release takes v5 to latest"
```

Comment:
```suggestion
title: "Express@5.1.0: Now the Default on npm with LTS Timeline"
```
This is just an idea, I think that we are doing 2 annoncements to the public (actually 3): New version (semver minor),  `latest` tag on v5 and finally the LTS big plan :smile: